_posts/2021-04-12-changelog-145.markdown

```diff
@@ -0,0 +1,89 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #145"
+date: 2021-04-12 17:00:00 +0300
+---
+
+
+## New Features
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6829 -->
+* Add `Unreachable code` inspection and quick-fix
+
+    TODO: GIF
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/7033 -->
+* Basic multiresolution at module scope in new [new name resolution](https://github.com/intellij-rust/intellij-rust/issues/6217) engine
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/7005 -->
+* Annotate wrong order of const generic parameters without `const_generics` feature
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6457 -->
+* Add inspection that annotates invalid references in `TOML` string literals. The inspection can be extended by other plugins. For example, Rust plugin provides file references to local dependency, build script and workspace paths in `Cargo.toml` as well as references to Cargo [features](https://doc.rust-lang.org/cargo/reference/features.html) and the inspection highlights all unresolved ones. Additionally, it provides a quick-fix to create a non-existent file from the corresponding file reference (by [@Kobzol])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6969 -->
+* Make dependencies completion in `Cargo.toml` more intuitive and faster when using crates local index. Note, this feature is disabled by default for now. To use it, enable `org.rust.crates.local.index` experimental feature
```

Comment:
let's move this item into fix section and group it with other fixes related to crates local index